<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN7P4A-0081" name="testscripts.YFPSimulator.MyChoice" tests="1" failures="0" timestamp="29 Jun 2017 08:12:49 GMT" time="6.740" errors="1">
  <testcase name="testRunner" time="6.740" classname="testscripts.YFPSimulator.MyChoice">
    <error type="com.cognizant.framework.FrameworkException" message="The specified file &quot;C:\Users\bpstestr\Desktop\CraftMaven\src\test\resources\Datatables\YFPSimulator.xls&quot; does not exist!">
      <![CDATA[com.cognizant.framework.FrameworkException: The specified file "C:\Users\bpstestr\Desktop\CraftMaven\src\test\resources\Datatables\YFPSimulator.xls" does not exist!
	at com.cognizant.framework.ExcelDataAccess.openFileForReading(ExcelDataAccess.java:86)
	at com.cognizant.framework.ExcelDataAccess.getRowNum(ExcelDataAccess.java:156)
	at com.cognizant.framework.ExcelDataAccess.getRowNum(ExcelDataAccess.java:231)
	at com.cognizant.Craft.DriverScript.initializeBusinessFlow(DriverScript.java:532)
	at com.cognizant.Craft.DriverScript.initializeTestScript(DriverScript.java:520)
	at com.cognizant.Craft.DriverScript.executeCraft(DriverScript.java:154)
	at com.cognizant.Craft.DriverScript.driveTestExecution(DriverScript.java:147)
	at testscripts.YFPSimulator.MyChoice.testRunner(MyChoice.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testRunner -->
</testsuite> <!-- testscripts.YFPSimulator.MyChoice -->
